obs-studio-small.json

```diff
@@ -14,6 +14,12 @@
             "OBS Studio (32bit)"
         ]
     ],
+    "persist": "config",
+    "post_install": [
```

Comment:
If you don't create the `config` (file or directory?) in the `post_install`, it won't persist (or does it already exist?)

If you don't list the `portable_mode` file in the `persist` tag, it won't persist across upgrades.

Reply:
config is a directory, which isn't created until OBS is run in portable mode. I forgot about adding portable_mode to persist.

Reply:
If the config directory doesn’t already exist, you’ll need to create it in the post_install, os it will persist across upgrades.